ci/lava/tests/test-boot-component-update.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python3
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""
+Pytest for testing boot component update.
+
+lorem ipsum
```

Comment:
What does this docstring mean?

Reply:
It means https://en.wikipedia.org/wiki/Lorem_ipsum :)

Reply:
Joking apart, it means that I forgot to add the docstring.